src/interpreter/ByteCodeInterpreter.cpp

```diff
@@ -537,6 +539,19 @@ Value ByteCodeInterpreter::interpret(ExecutionState& state, ByteCodeBlock* byteC
                 NEXT_INSTRUCTION();
             }
 
+            DEFINE_OPCODE(CallFunctionWithSpreadElement)
```

Comment:
Could you move down this byte code below?
Order of bytecode affects performance. (cpu code cache)
Only often used codes can be located on top